langwatch/prisma/schema.prisma

```diff
@@ -725,7 +725,7 @@ model LlmPromptConfig {
 
 model LlmPromptConfigVersion {
   id            String          @id @default(nanoid())
-  version       Int             @default(autoincrement())
+  version       Int             // The version number of the config
```

Comment:
This should get a migration, but it's not necessary for this PR as it's only a default and it's being handled automatically